src/components/30-organisms/footer/child.scss

```diff
@@ -20,8 +20,8 @@ axa-footer {
     text-transform: uppercase;
     line-height: 1.5;
 
-    @include breakpoint($mediaquery-xs-up) {
-      @include typo(primary, medium, bold);
+    @include breakpoint($mediaquery-sm-up) {
```

Comment:
You changed the behaviour from xs to sm. Because the breakpoint also changed or a mistake?

Reply:
done